src/Raven.Server/Documents/TransactionCommands/DeleteDocumentCommand.cs

```diff
@@ -56,9 +57,9 @@ public static DeleteDocumentCommand Deserialize(BlittableJsonReaderObject reader
         {
             if (!reader.TryGet(IdKey, out string id))
             {
-                return null;
+                throw new SerializationException($"Can't read {IdKey} of {nameof(DeleteDocumentCommand)}");
```

Comment:
`IdKey` should be replaced with `nameof`

Reply:
done